Apps/Playground/package.json

```diff
@@ -13,29 +15,35 @@
     "@babylonjs/core": "^5.0.0-alpha.6",
     "@babylonjs/loaders": "^5.0.0-alpha.6",
     "@babylonjs/react-native": "file:../../Modules/@babylonjs/react-native",
-    "@react-native-community/slider": "^2.0.9",
+    "@react-native-community/slider": "4.0.0-rc.2",
     "logkitty": "^0.7.1",
-    "react": "16.13.1",
-    "react-native": "0.63.1",
-    "react-native-permissions": "^2.1.4"
+    "react": "^17.0.1",
```

Comment:
@ryantrem are there any limitations to what react/react-native version we can use? What is required for testing when moving to a newer version?